mkdocs.yml

```diff
@@ -595,7 +595,7 @@ nav:
               - Git Repository Content: "development/apps/api/platform-features/git-repository-content.md"
               - Jinja2 Filters: "development/apps/api/platform-features/jinja2-filters.md"
               - Jobs: "development/apps/api/platform-features/jobs.md"
-              - Populating Extensibility Features: "development/apps/api/platform-features/populating-extensibility-features.md"
+              - Prepopulating Data: "development/apps/api/platform-features/prepopulating-data.md"
```

Comment:
Might be good to add a redirect from the old file name (around line 301 of this file)